src/main/java/com/sequenceiq/cloudbreak/shell/commands/ClusterCommands.java

```diff
@@ -128,7 +132,8 @@ public String createCluster(
                     stack, version, os,
                     stackRepoId, stackBaseURL,
                     utilsRepoId, utilsBaseURL,
-                    verify);
+                    verify,
```

Comment:
Can we make a check here if the security is enabled then check the corresponding values not to be null?